client/src/app.pcss

```diff
@@ -30,3 +30,22 @@
     opacity: 1;
   }
 }
+
+/*
+Webview styles
+*/
+
+@keyframes blink {
+  from {
+    background-color: gray;
+  }
+  to {
+    background-color: inherit;
+  }
+}
+
+.blink {
+  animation-name: blink;
+  animation-duration: 4s;
+  border-radius: 0.5rem;
+}
```

Comment:
We also have a blink animation in `SourceForm.svelte:412-429`. If you like you can replace the animation in SourceForm with this animation. I prefer the animation you defined more and this would avoid duplication.

Reply:
Done